tools/wheels/upload_wheels.sh

```diff
@@ -13,7 +15,7 @@ set_travis_vars() {
     else
       IS_PUSH="false"
     fi
-    if [[ "$TRAVIS_EVENT_TYPE" == "cron"  || -v CIRRUS_CRON ]]; then
+    if [[ "$TRAVIS_EVENT_TYPE" == "cron"  || -v "$CIRRUS_CRON" ]]; then
```

Comment:
Bash is clever like this, so you don't actually need to quote variables inside of double bracket tests. It lexes the `[[ ]]` before the variable is resolved to a string. :D

Quotes still don't cause any harm and some people stylistically prefer them instead of trying to figure out which quotes are or are not needed.

Reply:
Hrm, although it looks like you need the $ for:

```
if [[ $VAR == v* ]]; ...
```

So perhaps simpler to keep the quoting consistent?

Reply:
The other issue I guess is that the -v option to bash `[[` checks whether the following string is a variable name, so the check used to check whether `-v CIRRUS_CRON` -- i.e. is there a variable named CIRRUS_CRON. You don't want the dollar sign here, I suspect.

Reply:
The problem I was trying to resolve is:

```
    if [[ "$TRAVIS_EVENT_TYPE" == "cron"  || -v CIRRUS_CRON ]]; then
```

causing

```
tools/wheels/upload_wheels.sh: line 18: conditional binary operator expected
```

on MacOS.

Reply:
Hold on, I think I figured out the real problem here.

The problem is that `/bin/bash` on macOS is the bash that ships as part of Apple's image, which is the really really old one that predates February 2009. Installing bash from homebrew would get a modern bash that supports features which this script uses -- namely, the `-v` unary primary.

Reply:
:bulb: That should do it! I pushed an update, and also made consistent the quoting (I think).

Reply:
No luck, I'm afraid.